src/PhpSpreadsheet/Reader/Xlsx.php

```diff
@@ -1357,63 +1357,67 @@ public function load($pFilename)
+                                                    foreach ($styleArray as $stylePair) {
+                                                        $stylePair = explode(':', $stylePair);
 
-                                                    if ($stylePair[0] == 'margin-left') {
-                                                        $comment->setMarginLeft($stylePair[1]);
-                                                    }
-                                                    if ($stylePair[0] == 'margin-top') {
-                                                        $comment->setMarginTop($stylePair[1]);
-                                                    }
-                                                    if ($stylePair[0] == 'width') {
-                                                        $comment->setWidth($stylePair[1]);
-                                                    }
-                                                    if ($stylePair[0] == 'height') {
-                                                        $comment->setHeight($stylePair[1]);
-                                                    }
-                                                    if ($stylePair[0] == 'visibility') {
-                                                        $comment->setVisible($stylePair[1] == 'visible');
+                                                        if ($stylePair[0] == 'margin-left') {
+                                                            $comment->setMarginLeft($stylePair[1]);
+                                                        }
+                                                        if ($stylePair[0] == 'margin-top') {
+                                                            $comment->setMarginTop($stylePair[1]);
+                                                        }
+                                                        if ($stylePair[0] == 'width') {
+                                                            $comment->setWidth($stylePair[1]);
+                                                        }
+                                                        if ($stylePair[0] == 'height') {
+                                                            $comment->setHeight($stylePair[1]);
+                                                        }
+                                                        if ($stylePair[0] == 'visibility') {
+                                                            $comment->setVisible($stylePair[1] == 'visible');
+                                                        }
                                                     }
                                                 }
                                             }
                                         }
+                                    } else {
+                                        error_log('Invalid VML Comment XML File '.$fileWorksheet.'/'.$relPath);
```

Comment:
IMHO, if the xlsx file is invalid an exception should be thrown instead of an error being logged somewhere